### ISSUE TITLE
error starting emulator with --contracts

### ISSUE THREAD

**Issue by bjartek** [onflow/flow-cli#427]
### Problem

```flow emulator -v --contracts
INFO[0000] ⚙️   Using service account 0xf8d6e0586b0a20c7  serviceAddress=f8d6e0586b0a20c7 serviceHashAlgo=SHA3_256 servicePrivKey=0b29193872d2974c849abeacb16549c2d5cd8d1ccf4f7e2027bf19bf8c76e851 servicePubKey=9f7a868edeb24eff4848ae687c330021b493a4ddbfc2bea1f09fc2eeca5ff4b4e0d894cd81027a2d692544ab18b14c794a3f3689f6e507ff8ee361776c663855 serviceSigAlgo=ECDSA_P256
INFO[0000] 📜  Flow contract                              FlowFees=0xe5a8b7f23e8b548f
INFO[0000] 📜  Flow contract                              FlowStorageFees=0xf8d6e0586b0a20c7
INFO[0000] 📜  Flow contract                              FlowServiceAccount=0xf8d6e0586b0a20c7
INFO[0000] 📜  Flow contract                              FlowToken=0x0ae53cb6e3f42a79
INFO[0000] 📜  Flow contract                              FungibleToken=0xee82856bf20e2aa6
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x38 pc=0x10485de6c]

goroutine 1 [running]:
github.com/onflow/flow-go-sdk/crypto.InMemorySigner.Sign(...)
	github.com/onflow/flow-go-sdk@v0.24.0/crypto/crypto.go:129
github.com/onflow/flow-go-sdk.(*Transaction).SignEnvelope(0x14000234240, {0xf8, 0xd6, 0xe0, 0x58, 0x6b, 0xa, 0x20, 0xc7}, 0x0, ...)
	github.com/onflow/flow-go-sdk@v0.24.0/transaction.go:316 +0xe4
github.com/onflow/flow-emulator/server.deployContract(0x140006d6000, {0x1053d6c6d, 0x4}, {0x14000f26000, 0x1cdf, 0x2000})
	github.com/onflow/flow-emulator@v0.27.2/server/contracts.go:91 +0x284
github.com/onflow/flow-emulator/server.deployContracts(0x140006d6000)
	github.com/onflow/flow-emulator@v0.27.2/server/contracts.go:50 +0x384
github.com/onflow/flow-emulator/server.NewEmulatorServer(0x140000b5f10, 0x140000f2870)
	github.com/onflow/flow-emulator@v0.27.2/server/server.go:143 +0x87c
github.com/onflow/flow-emulator/cmd/emulator/start.Cmd.func1(0x1400036a780, {0x140005dae20, 0x0, 0x2})
	github.com/onflow/flow-emulator@v0.27.2/cmd/emulator/start/start.go:177 +0xb3c
github.com/spf13/cobra.(*Command).execute(0x1400036a780, {0x140001ba020, 0x2, 0x2})
	github.com/spf13/cobra@v1.2.1/command.go:860 +0x640
github.com/spf13/cobra.(*Command).ExecuteC(0x140001d2a00)
	github.com/spf13/cobra@v1.2.1/command.go:974 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
	github.com/onflow/flow-cli/cmd/flow/main.go:76 +0x4b8```

flow cli version 0.30.0, m1 macbook air, fresh `flow json file`


### Steps to Reproduce
```
flow version == 0.30
```mkdir a && cd a && flow init && flow emulator -v --contracts```

### Acceptance Criteria
It works

### Context

<what are you currently working on that this is blocking?>

**Comment by sideninja** [comment]
Update Flow CLI to latest version, this has been fixed in v0.30.1

**Comment by sideninja** [comment]
Should be fixed in latest version
